tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -334,8 +335,11 @@ def handler(event: Dict[str, Any], context: LambdaContext) -> None:
         raise ValueError(f"Must be passed a single record. Was {len(records)}")
```

Comment:
The default batch_size on `aws_lambda_event_source_mapping` is 10. I believe that conflicts with this limiter.